views/proxy_confirmation.pug

```diff
@@ -0,0 +1,14 @@
+extends layout
+
+block content
+  section.divider#request-info
+    h3= _("Oauth request for %s from: %s").format(kind, redirect_address)
+    div.panel-body
+          p.text-center
+            != _('Are you sure you would like to grant <a href="%s" target="_blank"> %s </a> access to the credentials of device: %s').format(redirect_address, redirect_address, kind)
```

Comment:
You cannot do this: if you use "!" you cannot do any string substitutions, otherwise you risk a script injection vulnerability.

Also, full URLs are dangerous from a usability POV - it's easy to make paths that look like domain names. We should probably display the domain name and port here only.